.travis.yml

```diff
@@ -46,9 +46,10 @@ before_install:
 #  - minikube update-context
 
 install:
-  - make setup-base
+  - make setup-dev
 
 script:
+  - make check
```

Comment:
I think this should be the last step, not the first. If my tests are failing I don't want to wait for linter every time. Can we make the linter run concurrently in another job somehow? How long does it take?

Reply:
It adds ~230 seconds to the build, but I've theoretically made check and test-ci run concurrently